### ISSUE TITLE
Unable to upload the example contract to testnet

### ISSUE THREAD

**Issue by njerschow** [osmosis-labs/osmosis-rust#17]
Hi,

I'm trying to compile the example contract (`examples/cosmwasm/contracts/osmosis-stargate`) but I run into issues during the upload step of deploying to testnet.

### Steps to reproduce:

```sh
cd examples/cosmwasm
RUSTFLAGS='-C link-arg=-s' cargo wasm
RES=$(osmosisd tx wasm store target/wasm32-unknown-unknown/release/osmosis_stargate.wasm --from main --gas-prices 0.1uosmo --gas auto --gas-adjustment 1.3 -y --output json -b block)
```

### Error Message:
```
Error: rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: 
Error calling the VM: Error during static Wasm validation: 
Wasm contract requires unsupported import: 
"__wbindgen_placeholder__.__wbindgen_describe". Required imports: 
{"__wbindgen_externref_xform__.__wbindgen_externref_table_grow", 
"__wbindgen_externref_xform__.__wbindgen_externref_table_set_null", 
"__wbindgen_placeholder__.__wbindgen_describe", ... 15 more}. 
Available imports: ["env.abort", "env.db_read", 
"env.db_write", "env.db_remove", "env.addr_validate", "env.addr_canonicalize", "env.addr_humanize", 
"env.secp256k1_verify", "env.secp256k1_recover_pubkey", "env.ed25519_verify", "env.ed25519_batch_verify", 
"env.debug", "env.query_chain", "env.db_scan", "env.db_next"].: create wasm contract failed: invalid request
```

**Comment by iboss-ptk** [comment]
Can you try `sha256sum artifacts/osmosis_stargate_example.wasm`

Here is mine

```
740d4b5a85b4804210b383a2c40e761bac6487d227e22c1f42d6d6f0b8c6fa15  artifacts/osmosis_stargate_example.wasm
```

And its deployed on testnet & initiated just fine, using the code from the main branch.

**Comment by njerschow** [comment]
Can you delete your artifacts folder and run this again? Your beaker command compiles my binary into `target/wasm32-unknown-unknown/release/osmosis_stargate.wasm` rather than `artifacts/osmosis_stargate_example.wasm`

Running `sha256sum target/wasm32-unknown-unknown/release/osmosis_stargate.wasm`

Gives:
```
1f7461d927ad1ac7e14d1d7ac5541072a3ea07d8615fe37db85405a5fa821f11  target/wasm32-unknown-unknown/release/osmosis_stargate.wasm
```

There is no artifact folder present (like you would get if you ran the cargo optimize command).

**Comment by iboss-ptk** [comment]
I suspect `cargo wasm`  = `cargo build --release --target=wasm32-unknown-unknown` ? I tried that and it produced `examples/cosmwasm/target/wasm32-unknown-unknown/release/osmosis_stargate.wasm` and with

```
RES=$(osmosisd tx wasm store target/wasm32-unknown-unknown/release/osmosis_stargate.wasm --from main --gas-prices 0.1uosmo --gas auto --gas-adjustment 1.3 -y --output json -b block)
```

It works properly as well.

I did `cargo clean` and delete artifacts folder.


Does `beaker wasm deploy` works?

**Comment by iboss-ptk** [comment]
This should be fixed by `v0.1.4` and related to #21, please open new issue if it's still a problem
